bad.js

```diff
@@ -0,0 +1,4 @@
+// no semicolon, bar is not defined, foo is never used
+var foo = bar
```

Comment:
'foo' is assigned a value but never used. (no-unused-vars)
'bar' is not defined. (no-undef)
Missing semicolon. (semi)